riverdriver/riverdatabasesql/migration/main/005_river_migration_add_line.up.sql

```diff
@@ -1,12 +1,18 @@
 ALTER TABLE river_migration
-    ADD COLUMN line text;
+    RENAME TO river_migration_old;
 
-UPDATE river_migration
-SET line = 'main';
+CREATE TABLE river_migration(
+    created_at timestamptz NOT NULL DEFAULT NOW(),
+    line TEXT NOT NULL,
+    version bigint NOT NULL,
```

Comment:
imo it's a little funky to not put the PK fields first just to make the default `SELECT * FROM river_migration` a bit cleaner. Seems like a warranted variance from the typical desire to alphabetize.

Reply:
Fair enough. Changed.